.travis.yml

```diff
@@ -14,4 +15,5 @@ script: vendor/bin/phpunit
 
 matrix:
   allow_failures:
+    - php: 7.0
```

Comment:
We shouldn't allow them to fail. We want to know if they're failing at this point.